internal/benthos/benthos-builder/builders/processors_test.go

```diff
@@ -23,7 +23,7 @@ func Test_buildProcessorConfigsJavascript(t *testing.T) {
 		Config: &mgmtv1alpha1.TransformerConfig{
 			Config: &mgmtv1alpha1.TransformerConfig_TransformJavascriptConfig{
 				TransformJavascriptConfig: &mgmtv1alpha1.TransformJavascript{
-					Code: `return "hello " + value;`,
+					Code: `return "hello " + value + " " + input.extra;`,
```

Comment:
this is a drive by change after working with a discord user. I noticed we don't actually have any tests that ensure the input works in the JS transformer processor.